### ISSUE TITLE
asm.bytes should not show bytes in Graph

### ISSUE THREAD

**Issue by Surendrajat** [radareorg/cutter#2240]
**Environment information**
* Operating System: elementary OS 5.1.5
* Cutter version: 1.10.3
* Obtained from:
  - [x] Built from source
  - [x] Downloaded from release
  - [ ] Distribution repository
* File format:

**Describe the bug**
There is no way to set `graph.bytes` in preferences. There's `asm.bytes` which affects both *Disassembly* and *Graph* views simultaneously.

**To Reproduce**

1. Go to **'Preferences -> Disassembly -> asm.bytes'** and enable it with `asm.nbytes` >= 1
2. Click on **Graph** view and see it enabled it here too.
3. Revert and see it gone.

**Expected behavior**
The **Graph** view should not be affected by `asm.bytes`. A setting can instead be given in **Graph** subsection of **Preferences** for `graph.bytes`.

**Screenshots**
![asm-bytes-graph](https://user-images.githubusercontent.com/11215000/84012566-5f298800-a995-11ea-9107-1677b3c752d1.gif)

**Additional context**

**Comment by Surendrajat** [comment]
Actually somewhat same for comments. Graph view can have its own preferences tab(not sub-tab/sub-section under Disassembly) and all related settings could be easily put there. Is there any reason to put Graphs under Disassembly as it is now?

**Comment by ITAYC0HEN** [comment]
> Is there any reason to put Graphs under Disassembly as it is now?
Yes and no.
Most of the configurations are not implemented separately for the graph. It inherits most `asm.*` configurations and thus you can't set many of these separately for graph and not for disassembly, or vice versa.

During the years, people added configuration for radare2's graph such as `graph.comments` or `graph.bytes` that can hold different values for disas and grpah. These values can be added to the Graph preferences on Cutter.

Bottom line, I don't have anything against having Graph settings outside of sub-item, especially because we are starting to give the users more and more control of the graph (e.g layout)

**Comment by Surendrajat** [comment]
Thanks, @ITAYC0HEN for the clarification. I'm new to this project and didn't use `r2` much either so I was wondering why graph view is considered sub-section of assembly. It will be better now to separate it in preferences too.
However, I now have new questions: whether `asm.bytes` will override `graph.bytes` if it's set in preferences or vice-versa? And how does this work with all the parameters that are in the graph as well?
